informus/rules/Delirium.md

```diff
@@ -100,7 +100,7 @@ NOTE:
 - the denominator can potentially be a bigger number than the current number of patients occupying the ICU beds (allowing for admissions / discharges), hence representing this as percentage and not a fraction
 - the following could potentially be excluded from the calculation:
 a) RASS -4/-5 with a CAM-ICU score - this is theoretically possible to document (by accident) but highly unlikely
-b) NO Valid RASS documented with a CAM-ICU score documented - this is possible as the bedside nurse may assume that a RASS score documented in the final hour of the last shift is still valid or incorrect as per clinical guidelines as RASS and CAM-ICU should be documented concurrently - both these scenarios are unlikely and would be even less likely once improvement work on this metric is underway
+b) NO Valid RASS documented with a CAM-ICU score documented means that at the time of CAM-ICU documentation, the last RASS score documented is now out of data (either it has been 1 hours and 15 minutes in the day or 4 hours and 15 minutes in the night). Ideally CAM-ICU and RASS should be documented concurrently as a concurrent RASS autofills one of the four CAM-ICU features, but this is not always done. Once improvement work on this metric is underway, this will be less likely. We will therefore need to denote when CAM-ICUs are documented without valid RASS score to prompt reviews of RASS on the floorplan. 
```

Comment:
```suggestion
b) NO Valid RASS documented with a CAM-ICU score documented means that at the time of CAM-ICU documentation, the last RASS score documented has now surpassed its validity period and become expired (either it has been 1 hours and 15 minutes in the day or 4 hours and 15 minutes in the night). Ideally CAM-ICU and RASS should be documented concurrently as a concurrent RASS autofills one of the four CAM-ICU features, but this is not always done. Once improvement work on this metric is underway, this will be less likely. We will therefore need to denote when CAM-ICUs are documented without valid RASS score to prompt reviews of RASS on the floorplan. 
```